packages/runtime/src/state/modules/edit-mode.ts

```diff
@@ -0,0 +1,25 @@
+import { Action, ActionTypes } from '../actions'
+
+export const EditModes = {
+  BUILD: 'build',
+  CONTENT: 'content',
+  INTERACT: 'interact',
+} as const
+
+export type EditMode = typeof EditModes[keyof typeof EditModes] | null
```

Comment:
`null` shouldn't be a part of this type, it seems to be a part of the state not the underlying enum